### ISSUE TITLE
Stream initialization error (OpeningDevice)

### ISSUE THREAD

**Issue by Personuo** [keijiro/Lasp#54]
When I run DeviceSelector scene, I immediately get this error Stream initialization error (OpeningDevice)

win10.
unity version 2021.3.4

Not sure if I'm getting something wrong, I'm not using any midi input device, I'm just trying to get the audio from loopback on windows.

I just download and run, no do anything

**Comment by keijiro** [comment]
What audio device are you using?

**Comment by Personuo** [comment]
A normal realtek(R) Audio in my laptop integrated sound card.
I've tried cscore and fmod with no problems,both can detect the device and output spectrum data

In Lasp, the device is still found normally , although the error is always output. The device drop-down has my sound devices (both microphone and realtek(R) Audio).

**Comment by keijiro** [comment]
Is there any other processes (apps) using that audio device? Windows usually doesn't allow sharing an audio device between multiple processes.

**Comment by Personuo** [comment]
There should be none, when I restarted my computer and tried lasp immediately, I still get the error.
Then I am running the cscore and fmod projects and they still work well

**Comment by Personuo** [comment]
I mean, I don't have any external audio devices, only the sound card in the motherboard of my laptop.

My expectation is to play youtube music in the chrome and then unity can receive the data of this audio from loopback. (The music can still be played from my laptop speakers)

**Comment by keijiro** [comment]
What happens when switching "Use Default Device"?

By the way, you can't do loopback with LASP. You have to install a virtual audio device app to loop it back to LASP.

**Comment by Personuo** [comment]
yes, I found another desktop windows to test and it works fine but only searches for the microphone.
If so, then I will try other tools, thank you

**Comment by delgelato** [comment]
Hi
I'm getting this error too. Worked on my Unity project (just adding Odin), then got this error, then reverted back to previous commit, and the error's still spamming
Tried the same project on another comp and it works fine
Could it be... windows update related or an audio device breaking? Changed to any device and also default, and on a fresh Lasp project, and the issue's still there
I'm wondering if there's a way to "hard reset" all windows audio device and hopefully fixes this?

Cheers

**Comment by CharlesGrs** [comment]
@keijiro Hey Kejiro, same issue here; I have been using LASP for a while now without any issue I have no idea what causes this error now.. I don't think there is any other software using the audio device any idea on how to track that issue ?

Few more information, Unity 2023.1.16 (it was working with that version previously), on Windows 11, absolutely nothing else open, and same error for every device that I select, no matter the device .. 

Many thanks

**Comment by CharlesGrs** [comment]
This issue was solved by giving Unity access to the microphone in the privacy settings of Windows 11.
